core/src/Cabana_CommunicationPlan.hpp

```diff
@@ -588,9 +588,10 @@ class CommunicationPlan
       this rank, just use this rank as the export destination and the data
       will be efficiently migrated.
     */
-    template <class ViewType>
-    Kokkos::View<size_type*, device_type>
-    createFromExportsAndTopology( const ViewType& element_export_ranks,
+    template <class ExecutionSpace, class ViewType>
+    Kokkos::View<size_type*, memory_space>
+    createFromExportsAndTopology( ExecutionSpace,
```

Comment:
When we refactor further, we need to use this execution space to construct execution policies

Reply:
Created an issue